src/pages/RepoPage/FailedTestsTab/FailedTestsTab.tsx

```diff
@@ -109,7 +106,63 @@ export default function FailedTestsTab() {
         </p>
       </div>
       <SetupOptionSelector />
-      <Content />
+      {usesCli ? <CodecovCLI /> : <GitHubActions />}
     </div>
   )
 }
+
+const Loader = () => (
+  <div className="flex justify-center py-8">
+    <Spinner />
+  </div>
+)
+
+interface URLParams {
+  provider: string
+  owner: string
+  repo: string
+}
+
+function FailedTestsTab() {
+  const { provider, owner, repo } = useParams<URLParams>()
+  const { data: repoOverview } = useRepoOverview({ provider, owner, repo })
+
+  if (repoOverview?.testAnalyticsEnabled) {
+    return (
+      <SentryRoute
```

Comment:
Can we add a redirect from` /tests/:branch/*` to `/tests/:branch` here?

Reply:
Updated in https://github.com/codecov/gazebo/pull/3042/commits/18984d4a734599b3a37b2fc89de0d9a269d73d90